.circleci/config.yml

```
@@ -1,5 +1,8 @@
 version: 2.1
 
+orbs:
+  code-infra: https://raw.githubusercontent.com/mui/mui-public/refs/heads/ci-orb/.circleci/orbs/code-infra.yaml
```

To be changed to master before merging

```suggestion
  code-infra: https://raw.githubusercontent.com/mui/mui-public/refs/heads/master/.circleci/orbs/code-infra.yaml
```